grumblr/app/controllers/application_controller.rb

```diff
@@ -2,4 +2,14 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
-end
+
+    def set_access_control_headers
```

Comment:
sorry, i forgot to delete it from the other version.

On Thu, May 21, 2015 at 11:51 AM, Adam Bray <notifications@github.com>
wrote:

> In grumblr/app/controllers/application_controller.rb
> <https://github.com/ga-dc/grumblr/pull/21#discussion_r30816364>:
>
> > @@ -2,4 +2,14 @@ class ApplicationController < ActionController::Base
> >    # Prevent CSRF attacks by raising an exception.
> >    # For APIs, you may want to use :null_session instead.
> >    protect_from_forgery with: :exception
> > -end
> > +
> > +    def set_access_control_headers
>
> We don't need this method technically. Since our JS (backbone) is served
> from the same server (rails), we don't need to enable CORS.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-dc/grumblr/pull/21/files#r30816364>.
>



-- 
Keith W. Knox